docs/content/charts/configs/PieChart.js

```diff
@@ -1,19 +1,33 @@
-import {Md} from 'docs/components/Md';
+import { Md } from 'docs/components/Md';
 
 import boundedObject from '../../svg/configs/BoundedObject';
 
 export default {
-   ...boundedObject,
-   angle: {
-      key: true,
-      type: 'number',
-      description: <cx><Md>
-         Angle in degrees. Default is `360` which represents the full circle.
+    ...boundedObject,
+    angle: {
+        key: true,
+        type: 'number',
+        description: <cx><Md>
+            Angle in degrees. Default is `360` which represents the full circle.
       </Md></cx>
-   },
-   class: false,
-   style: false,
-   preserveWhitespace: false,
-   trimWhitespace: false,
-   plainText: false
+    },
+    startAngle: {
+        key: true,
+        type: 'number',
+        description: <cx><Md>
+            Start angle in degrees. Indicates the starting point of the first stack. Default is `0`.
+      </Md></cx>
+    },
+    clockWise: {
```

Comment:
Please rename to `clockwise` since it's a single word. camelCase is commonly used to concatenate multiple words.

Reply:
Done.